Lesson24/Ayse/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Posts</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>Posts</h1>
+      <button id="fetch-posts" class="button button--success">
+        Fetch all posts
+      </button>
+
+      <button id="create-post-btn" class="button button--success">
+        Create New Post
+      </button>
+    </div>
+
+    <ul id="posts-container"></ul>
+
+    <script>
+      document
```

Comment:
Make the 'Create New Post' an <a> element and remove the JS code added here directly.